### PR TITLE
va-segmented-progress-bar: Hiding steps from JAWS

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://2209-hide-steppers-aria--60f9b557105290003b387cd5.chromatic.com

---

## Description
Closes [#2209](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2209)

## Testing done
Tested locally using storybook, brave, and VoiceOver. Did not have JAWS to test with.

## Screenshots
N/A

## Acceptance criteria
- [ ] Does not repeat the step indicator text in JAWS

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by Andrew565** [comment]
@jeana-adhoc Please test this with JAWS at your earliest convenience and let me know if it meets expectations for this ticket, thanks!

File: packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx [hunk 1]
```diff
@@ -185,6 +185,7 @@ export class VaSegmentedProgressBar {
               aria-valuemin="0"
               aria-valuemax={total}
               aria-valuetext={label}
+              aria-hidden="true"
             >
```
[the remark this answers is not in the document]
Comment (Andrew565) [reply packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx]: Done

**Comment by Andrew565** [comment]
@jeana-adhoc Friendly reminder to review this at your earliest convenience with JAWS, thanks!

**Review by jeana-adhoc — APPROVED** [review]
I tested this in JAWS with the storybook link provided and it no longer repeats like it did before.

**Review by nickjg231 — APPROVED** [review]
LGTM

**Pull request merged**